nrf_wifi/doc/fmac.rst

```diff
@@ -0,0 +1,12 @@
+FMAC
```

Comment:
```suggestion
.. _fmac_module:

FMAC
```
All docs need to have a name tag so that you can link to them. The name I used here is just a suggestion.